adafruit_ccs811.py

```diff
@@ -156,6 +158,31 @@ def _update_data(self):
             if self.error:
                 raise RuntimeError("Error:" + str(self.error_code))
 
+    @property
+    def baseline(self):
+        """
+        The function read and return the current baseline value.
+        The returned value is packed into an integer.
+        Later the same integer can be passed `set_baseline` function
+        in order to set a new baseline.
+        """
+        buf = bytearray(3)
+        buf[0] = _BASELINE
+        with self.i2c_device as i2c:
+            i2c.write_then_readinto(buf, buf, out_end=1, in_start=1)
+        return struct.unpack(_BASELINE_PACKING_FORMAT, buf[1:])[0]
+
+    def set_baseline(self, baseline_int):
```

Comment:
Here is the change for what @ladyada is suggesting.
```suggestion
    @baseline.setter
    def baseline(self, baseline_int):
```

Reply:
thank you, updated